rover_msgs/Obstacle.lcm

```diff
@@ -3,4 +3,5 @@ package rover_msgs;
 struct Obstacle {
 	double bearing; // from straight ahead
 	double distance; // from straight ahead
+	double rightBearing;
 }
```

Comment:
Can you add a line at the end of this file?

Reply:
Fixed